src/mlx/warnings/warnings.py

```diff
@@ -20,7 +20,7 @@
 from .regex_checker import CoverityChecker, DoxyChecker, SphinxChecker, XMLRunnerChecker
 from .robot_checker import RobotChecker
 
-__version__ = distribution("mlx.warnings").version
+__version__ = version("mlx-warnings")
```

Comment:
For future reference: `importlib.metadata` was under development since Python 3.8 and finalized in 3.10. The convenience function that converts the `.` to a `-` automatically was introduced in the final stage of development.